content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,157 @@
+But how does the computer know how many cycles there are in a second?
+We can't actually wait for a second, because the delay (also called latency) would be very noticeable and make the system unsuitable for scratching.
+
+## Analog vs. Digital Audio
+
+It's important to know that in contrast to analog audio like vinyl or music cassettes, the computer doesn't actually work with the continuous wave that we see in the screenshot.
+
+![Analog Signal]({static}/images/news/signal_analog.svg)
+
+Instead, [digital signals](https://en.wikipedia.org/wiki/Digital_signal_%28signal_processing%29) work with measurements (called "samples") that indicate the value of the wave position at specific points in time.
+How often these measurements are taken is determined by the sample rate (or sampling rate).
+For example, Audio CDs use a sample rate of 44100 Hz (i.e. 44100 measurements per second, or 1 measurement every ~22.68 microseconds).
+Such a signal is called "time-discrete":
+
+![Time-Discrete Signal]({static}/images/news/signal_time_discrete.svg)
+
+The reason for this is that a time-continuous signal has infinite resolution and would need infinite memory space to save, which is impossible - and also unnecessary, because as long as you take enough samples per second, you can [restore the original signal](https://en.wikipedia.org/wiki/Nyquist%E2%80%93Shannon_sampling_theorem)  without any loss of information.
+
+Since computers can't store numbers with a theoretically infinite number of different values, a digital signal is also value-discrete.
+This means that the amplitude values are mapped to a finite number of different values.
+
+Let's assume we only have space to store 11 distinct values per sample (-1.0, -0.8, -0.6, -0.4, -0.2, 0, 0.2, 0.4, 0.6, 0.8 and 1.0).
+But a sample value may have any value, so if its value would be between two of these values, they have to be rounded (in this case: multiples of 0.2).
+This process is called [Quantization](https://en.wikipedia.org/wiki/Quantization_%28signal_processing%29) and causes some information loss, which means that it [adds noise](https://en.wikipedia.org/wiki/Quantization_%28signal_processing%29#Error) to our signal.
+An example for such a quantized, value-discrete version of the analog signal looks like this:
+
+![Value-Discrete Signal]({static}/images/news/signal_value_discrete.svg)
+
+A more realistic example is the Audio CD, which uses 16 bit per sample value.
+With 16 bits, you can represent 2<sup>16</sup> = 65536 different values.
+
+A *digital* signal is both time-discrete and value-discrete:
+
+![Digital Signal]({static}/images/news/signal_digital.svg)
+
+Note that this is just a very basic introduction in how a digital signal is represented.
+If you're interested in the topic, I recommend watching [Monty Montgomery's "Digital Show & Tell" video](https://xiph.org/video/vid2.shtml), where he demonstrates how digital signals behave in contrast to analog ones and clears up common misconceptions.
```

Comment:
```suggestion
If you're interested in the topic, I recommend watching [Monty Montgomery's "Digital Show & Tell" video](https://xiph.org/video/vid2.shtml), where he demonstrates how digital signals behave in contrast to analog ones and where he also clears up common misconceptions.
```
misconceptions on what exactly? 
also adding some extra redundant words here reduces the chance of tripping up there imo.

Reply:
> misconceptions on what exactly?

On how digital signals behave in contrast to analog ones.

Reply:
Could you add that to the sentence in question?

Reply:
I don't know how without repetition. The current text already says:

> he demonstrates how digital signals behave in contrast to analog ones and clears up common misconceptions.